config/settings.yml

```diff
@@ -36,6 +36,7 @@ directories:
 # Storage options
 storage:
   track_revisions: false
+  notebook_file_class: true
```

Comment:
This should actually be set to false by default, I will change that before the final merge.